packages/react-i18n/src/index.tsx

```diff
@@ -12,18 +13,65 @@ export interface I18nReact {
 	_x: I18n[ '_x' ];
 	isRTL: I18n[ 'isRTL' ];
 	i18nLocale: string;
+	localeData?: LocaleData;
+	hasTranslation: Function;
+	addFilter?: Function;
+	removeFilter?: Function;
+	applyFilters?: Function;
```

Comment:
The `*Filters` functions will be also exposed by the `useI18n` hook? Is that fine? Why do we need the extra `useFilters` hook?

Reply:
`useFilters` has been removed.

> The `*Filters` functions will be also exposed by the `useI18n` hook? Is that fine?
I don't see a problem. Do you have anything specific in mind?